aerospike-mixin/dashboards/aerospike-overview.libsonnet

```diff
@@ -0,0 +1,1176 @@
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [
+      {
+        matcher: {
+          id: 'byName',
+          options: 'Value',
+        },
+        properties: [
+          {
+            id: 'custom.hidden',
+            value: true,
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'Time',
+        },
+        properties: [
+          {
+            id: 'custom.hidden',
+            value: true,
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'instance',
+        },
+        properties: [
+          {
+            id: 'custom.filterable',
+            value: true,
+          },
+          {
+            id: 'links',
+            value: [
+              {
+                title: 'Instance overview',
+                url: '/d/aerospike-instance-overview',
+              },
+            ],
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'aerospike_cluster',
+        },
+        properties: [
+          {
+            id: 'displayName',
+            value: 'Aerospike cluster',
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'ns',
+        },
+        properties: [
+          {
+            id: 'displayName',
+            value: 'namespace',
+          },
+        ],
+      },
+    ],
+  },
+  links: [
+    {
+      title: 'Namespace overview',
+      url: '/d/aerospike-namespace-overview',
+    },
+  ],
+  options: {
+    cellHeight: 'md',
+    footer: {
+      countRows: false,
+      fields: '',
+      reducer: [
+        'sum',
+      ],
+      show: false,
+    },
+    showHeader: true,
+    sortBy: [],
+  },
+  pluginVersion: '10.2.0-59542pre',
+  transformations: [
+    {
+      id: 'sortBy',
+      options: {
+        fields: {},
+        sort: [
+          {
+            desc: true,
+            field: 'Time',
+          },
+        ],
+      },
+    },
+  ],
+};
+
+local unavailablePartitionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(job, aerospike_cluster) (aerospike_namespace_unavailable_partitions{job=~"$job", aerospike_cluster=~"$aerospike_cluster"})',
+      datasource=promDatasource,
+      legendFormat='{{aerospike_cluster}}',
+      format='time_series',
+    ),
+  ],
+  type: 'bargauge',
+  title: 'Unavailable partitions',
+  description: 'Number of unavailable data partitions in an Aerospike cluster.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      decimals: 0,
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
```

Comment:
I think you could add in a threshold here for 1 similar to other unavailable partition panels throughout the mixin, unless i'm missing something and it doesn't make sense here

Reply:
yeah, if it's really bad, then we should have anything thats > 0 show up as red maybe? for both partition panels